test/report.test.ts

```diff
@@ -88,9 +88,13 @@ describe('report', () => {
 
                 return mockedReportModule()
                     .exportResult(projectId, executionResult, exportConfig, { pollStep: 1 })
-                    .then(null, (error: Error) => expect(error.message).toBe(
-                        'You cannot export this insight because it contains restricted data.'
-                    ));
+                    .then(null, (error: ApiResponseError) => {
+                        expect(error.response.status).toBe(400);
+                        expect(error.response.statusText).toBe('Bad Request');
+                        expect(error.message).toBe(
+                            'You cannot export this insight because it contains restricted data.'
```

Comment:
it should be `.toBe(ERROR_RESTRICTED_MESSAGE)`

Reply:
updated :)